tests/unit/repository/test_base_repo.py

```diff
@@ -980,3 +980,25 @@ def test_get_tags_with_no_results(memory_repository):
     )
 
     assert tags == []
+
+
+def test_add_comments(memory_repository):
+    repository = memory_repository
+    experiment = _create_experiment(repository)
+    repository.add_comments(
+        experiment.project_name,
+        ["wow"],
```

Comment:
can we make this value a few words long? just wanna validate that its not splitting on spaces or anything like that